ui/app/components/actions/context.ts

```diff
@@ -8,9 +8,9 @@ export default class ActionsRelease extends Component {
   @action
   toggleHint() {
     if (this.hintIsVisible === true) {
-      return this.hintIsVisible = false;
+      return (this.hintIsVisible = false);
     } else {
-      return this.hintIsVisible = true;
-    };
+      return (this.hintIsVisible = true);
```

Comment:
The fact that we're returning assignments here is weird, and wrapping those in expressions makes it look even weirder. Maybe we should be using `mut` or the `set/pick` helpers instead ? (not in this PR though)

Reply:
Yeah, I agree, it is odd. As you say, something we can fix in a different PR.

Reply:
That aside, are you happy to proceed with these changes?